src/app/learnocaml_partition_view.ml

```diff
@@ -174,13 +223,28 @@ let main () =
       then update_repr_code id
       else true in
 
-  retrieve (Learnocaml_api.Partition (teacher_token, exercise_id, fun_id, prof))
-  >>= fun part ->
+  let fetch_students =
+    retrieve (Learnocaml_api.Students_list teacher_token)
+    >>= fun students_map ->
+    let students = students_map in
+    Lwt.return students
```

Comment:
Hint: these three lines just implement the identity: can be removed… while keeping the code semantics :)

```suggestion
```